pkg/datapath/linux/node.go

```diff
@@ -1213,8 +1213,7 @@ func (n *linuxNodeHandler) NodeConfigurationChanged(newConfig datapath.LocalNode
 				return fmt.Errorf("direct routing device is required, but not defined")
 			}
 
-			var targetDevices []string
-			targetDevices = append(targetDevices, option.Config.DirectRoutingDevice)
+			targetDevices := []string{option.Config.DirectRoutingDevice}
```

Comment:
Any reason not to use `make` with the length of `option.Config.GetDevices()` + 1 here? (Note I'm a Go newbie.)